tests/admin_checks/tests.py

```diff
@@ -913,3 +913,13 @@ class BookAdminWithListFilter(admin.ModelAdmin):
             self.assertEqual(errors, [])
         finally:
             Book._meta.apps.ready = True
+
+    def test_inlines_property(self):
+        class CitiesInline(admin.TabularInline):
+            model = City
+
+        class StateAdmin(admin.ModelAdmin):
+            inlines = [CitiesInline]
```

Comment:
1d8eb0cae57731b481a88dca272b2cb0d645bd8e fixed admin checks for properties, so we should test here:
```python
@property
def inlines(self):
    return [CitiesInline]
```
I'm going to update this test.